nexus_demos/config/pick_and_place.json

```
@@ -1,18 +1,33 @@
 {
   "workInstructionName": "Pick and Place",
-  "metadata": {
-    "SkuId": "productA",
-    "description": "productA",
-    "unit": "dummy_unit",
-    "quantity": 1.0,
-    "quantityPerPallet": 1.0
-  },
   "steps": [
     {
-      "processId": "place_on_conveyor"
+      "processId": "place_on_conveyor",
+      "outputItems": [
+        {
+          "guid": "productA",
+          "metadata": {
+            "description": "productA",
+            "unit": "dummy_unit",
+            "quantity": 1.0,
+            "quantityPerPallet": 1.0
+          }
```

To make these work orders less verbose... (also for other files)

```suggestion
          "guid": "productA",
          "metadata": {
            "quantity": 1,
            "state": "state_1"
          }
```

---

https://github.com/osrf/nexus/pull/87/commits/3ff960d2e8b7751f55b025747856d883cf653a53